docs/toolsets.md

```diff
@@ -0,0 +1,46 @@
+# Toolsets
+
+PydanticAI ships with native tools and toolsets (collections of tools) that can be used to enhance your agent's capabilities.
```

Comment:
```suggestion
```
I'd suggest we don't say this yet considering we don't have any others lol. Once we do we can make a comment about supporting more? Also, there are no "toolsets" now, right?

Reply:
There's no toolset yet.